.NET/Microsoft.Recognizers.Definitions.Common/French/DateTimeDefinitions.cs

```diff
@@ -43,9 +43,9 @@ public static class DateTimeDefinitions
       public const string WrittenOneToNineRegex = @"(?:une?|deux|trois|quatre|cinq|six|sept|huit|neuf)";
       public const string WrittenElevenToNineteenRegex = @"(?:(seize|quinze|quatorze|treize|douze|onze)|dix\W(neuf|huit|sept))";
       public const string WrittenTensRegex = @"(?:quatre\Wvingt(s|\Wdix)?|soixante(\Wdix)?|dix|vingt|trente|quarante|cinquante|septante|octante|huitante|nonante)";
-      public static readonly string WrittenCenturyFullYearRegex = $@"(?:(deux\s+)?mille((\s+{WrittenOneToNineRegex})?\s+cents?)?)";
+      public static readonly string WrittenCenturyFullYearRegex = $@"(?:(deux\s+)?mille((\s+{WrittenOneToNineRegex})?\s+cent?)?)";
```

Comment:
Why this change? The s? is already saying the s is optional, useful for cases where you could have a duration of multiple centuries.

Reply:
> Why this change? The s? is already saying the s is optional, useful for cases where you could have a duration of multiple centuries.

In French, when express century and decade, cents is incorrect, it should be cent.

Reply:
> > Why this change? The s? is already saying the s is optional, useful for cases where you could have a duration of multiple centuries.
> 
> In French, when express century and decade, cents is incorrect, it should be cent.

Since s is optional, it should be fine to keep it, I will revert the change.